Library/Homebrew/extend/os/linux/keg_relocate.rb

```diff
@@ -44,11 +41,9 @@ def change_rpath(file, old_prefix, new_prefix)
     else
       old_interpreter.sub old_prefix, new_prefix
     end
-    cmd << "--set-interpreter" << new_interpreter if old_interpreter != new_interpreter
+    updated[:interpreter] = new_interpreter if old_interpreter != new_interpreter
 
-    return if old_rpath == new_rpath && old_interpreter == new_interpreter
-
-    safe_system(*cmd, file)
+    file.patch!(**updated)
```

Comment:
This will break when we upgrade to Ruby 2.7. Please instead do:

```suggestion
    file.patch!(interpreter: interpreter, rpath: rpath)
```